.github/workflows/ci.yml

```diff
@@ -34,6 +34,8 @@ jobs:
         uses: taiki-e/install-action@cargo-llvm-cov
 
       - name: Generate code coverage
+        env:
+          RUST_BACKTRACE: "1"
```

Comment:
Did you mean to keep this enabled here?

Reply:
I was thinking so; more useful info when something does go wrong (especially if it doesn't repro locally).